dashboard/lib/service/dev_cocoon.dart

```diff
@@ -274,10 +274,11 @@ class DevelopmentCocoonService implements CocoonService {
     final int minAttempts = _minAttempts[status];
     final int maxAttempts = _maxAttempts[status];
     final int attempts = minAttempts + random.nextInt(maxAttempts - minAttempts + 1);
+    final int randomQueueTime = random.nextInt(1000 * 60 * 15);
     final Task task = Task()
       ..createTimestamp = Int64(commitTimestamp + index)
-      ..startTimestamp = Int64(commitTimestamp + index + 10000)
-      ..endTimestamp = Int64(commitTimestamp + index + 10000 + random.nextInt(1000 * 60 * 15))
+      ..startTimestamp = Int64(commitTimestamp + index + randomQueueTime)
+      ..endTimestamp = Int64(commitTimestamp + index + randomQueueTime + random.nextInt(1000 * 60 * 15))
```

Comment:
Another approach would be to do 

```
      ..startTimestamp = Int64(commitTimestamp + (index * 1000 * 60))
```

Reply:
Thanks for the reply! Would it be better to change random.nextInt(1000 * 60 * 15) in endTimestamp as well? like would having `..endTimestamp = Int64(commitTimestamp + random.nextInt(1000 * 60 * 15) + random.nextInt(1000 * 60 * 15))` be better, or is endTimestamp encouraged to have more randomness encoded?

Reply:
Yes, let's remove random from endTimestamp as well. The more deterministic fake data is, the better. Otherwise, it's an easy way to introduce flakes to our CI.